test/image.bats

```diff
@@ -293,3 +293,27 @@ function teardown() {
 
 	cleanup_images
 }
+
+@test "run container in pod with crun-wasm enabled" {
+	cat << EOF > "$CRIO_CONFIG_DIR/99-crun-wasm.conf"
+[crio.runtime.runtimes.crun-wasm]
+runtime_path = "/usr/bin/crun"
+
+[crio.runtime.runtimes.crun-wasm.arch_runtime_paths]
+wasm = "/usr/bin/crun-wasm"
+EOF
+	start_crio
+
+	# Assuming the $IMAGE_LIST_DIGEST environment variable contains the new image value
+	IMAGE_LIST_DIGEST="quay.io/sohankunkerkar/test-wasm:latest"
```

Comment:
keeping this for testing purposes. I will move it to `quay.io/cri-o` once the test executes successfully.